src/main/java/Duke.java

```diff
@@ -1,10 +1,84 @@
+import duke.exception.DukeException;
+import duke.iomethods.IOMethods;
+import duke.task.TaskManager;
+import duke.fileHandler.FileHandler;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
 public class Duke {
-    public static void main(String[] args) {
+
+    //fields
```

Comment:
This comment is not very helpful in providing further explanation on how the variable is used. It can be omitted.